GTMAppAuth/Sources/KeychainStore/KeychainStore.swift

```diff
@@ -101,10 +102,20 @@ public final class KeychainStore: NSObject, AuthSessionStore {
   @objc(saveAuthSession:error:)
   public func save(authSession: AuthSession) throws {
     let authSessionData: Data = try authSessionData(fromAuthSession: authSession)
+
+    var maybeAccessibility: CFString? = kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly
+    if #available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *) {
+#if os(macOS)
```

Comment:
`#if` is a compile-time option, `if #available` is evaluated at runtime. Wouldn't it be better to just have
```swift
if #available(macOS 10.15, *), !keychainAttributes.contains(.useDataProtectionKeychain) {
    // accessibility options on macOS only work when using the data protection keychain
    maybeAccessibility = nil
}
```